docs/spec_draft.md

```diff
@@ -850,6 +851,42 @@ implementation-defined.
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.exponential_minus_one
+
+### Semantics
+
+Performs element-wise exponential minus one operation on `operand` tensor and
+produces a `result` tensor. For floating-point element types, it implements the
+`expm1` operation from the IEEE-754 specification. For complex element types, it
+computes a complex exponential minus one, with corner cases TBD. Numeric
```

Comment:
The PowOp is similar, I did not mention anything about corner cases, not sure if it is required or mention of IEEE spec although precision being implementation defined is correct to mention. Complex exponential is well defined mathematically.

Reply:
Let me know your thoughts, PowOp is similar in some regards

Reply:
I am thinking why should we link IEEE spec for simple ops such as these, but we have done that for other ops such as addition etc, so we can refer the spec for consistency.

Reply:
> Let me know your thoughts, PowOp is similar in some regards

Complex exponential is well defined, but still need to mention it here since it is not defined in IEEE-754, and we have been following this convention for other similar elemental ops thus written here for consistency.

> I am thinking why should we link IEEE spec for simple ops such as these

We are aiming to follow a standard for ops that require special handling such as numeric precision, error handling, corner cases, etc., and IEEE-754 is a well-established one that has defined most, if not all, of these behaviors.